core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -836,6 +837,14 @@ public Integer getSchedulerPollIntervalSecond() {
         return Integer.parseInt(getOptional("kylin.job.scheduler.poll-interval-second", "30"));
     }
 
+    public boolean isSchedulerSafeMode() {
+        return Boolean.parseBoolean(getOptional("kylin.job.scheduler.safemode", "false"));
+    }
+
+    public List<String> getSafeModeRunableProjects() {
+        return Arrays.asList(getOptionalStringArray("kylin.job.scheduler.safemode.runable-projects", null));
```

Comment:
Typo of "runable" here, typos in config items should not exist

Reply:
fixed. Please help to review again.